examples/ipv6-eks-cluster/README.md

```diff
@@ -1,11 +1,11 @@
-# EKS Cluster with Amazon EKS Managed Add-ons
```

Comment:
Maybe I missed it, but why are we replacing the `EKS Cluster with Amazon EKS Managed Add-ons` example with IPv6 and not adding a new example?

Reply:
IPv6 has its own example now. I removed the Amazon EKS Managed Add-ons example and replaced with `complete-kubernetes-addons` example now.